install.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+###############################################################
+#                       DEPLOY SCRIPT                         #
+#                                                             #
+#          Script used for creating the specific              #
+#        tag used for triggering the CI deployment            #
+#                                                             #
+#               End tag should look like this:                #
+#                  internal-all/v1.0.0-1234                   #
+#                                                             #
+#                Prepared by Jasmin Abou Aldan                #
+#       Copyright (c) 2020 Infinum. All rights reserved.      #
+###############################################################
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+echo "==> ${bold}This script will install:${normal}"
+echo "/usr/local/bin/app-deploy"
+echo
+read -r -p "Do you want to proceed? [y/n] " c
+if ! [[ ${c} =~ ^(yes|y|Y) ]] || [ -z ${c} ]; then
+    exit 1
+fi
+echo
+echo "Fetching script data..."
+mkdir .tmp
```

Comment:
Rather use some name like `.app_deploy_tmp` because tmp can be on users system and potentially you would end up with deleting its data.